### ISSUE TITLE
Position is invalid if drop target contains elements

### ISSUE THREAD

**Issue by mdevlamynck** [norpan/elm-html5-drag-drop#24]
I'm trying to implement a reorderable list and I'm using `Position` to find if the drop needs to happen above or bellow the drop zone (which is another element of the list). The issue is that `offsetX` and `offsetY` read in the event to build `Position` is the relative position of the mouse to the DOM node the mouse is hovering and not the `droppable` DOM node.

[I slightly modified the example Ellie to show the issue.](
https://ellie-app.com/pjbym3Xt7rLa1)

When you try to drop the elm logo on the bottom of the red square, the dropzone background is cyan instead of purple. If you move the mouse horizontally, as soon as you exit the red square the background is purple. This is because in the bottom of the red square the `offsetY` is still smaller than `currentTarget.clientHeight / 2` because that offset is relative to the red square and not the dropzone.

Is there a way to ignore child nodes of `droppable` nodes?

**Comment by norpan** [comment]
Yeah, this is an issue I don't really know how to solve, unfortunately.

The problem is that Elm provides no good way to get access to anything else than the parameters of the event itself. What you'd want to do is to call getBoundingClientRect(), but I haven't find a good way to do that. The only way to interact with javascript is through ports, and you can't really call a port here.

**Comment by mdevlamynck** [comment]
That's what I was afraid of. Maybe I can try to workaround this issue with z-index. Seems to work in some case but it's finicky. Thx for your quick answer :)

**Comment by mdevlamynck** [comment]
Adding `pointer-events: none` on the children fixes the issue : https://ellie-app.com/pjcHxYdKbgKa1. I'm not sure how it could be integrated in pure elm in this lib though. At least it works for my needs. (I only apply it when dragging to still allow interactions in the children when not dragging). Maybe it could be added to the docs at least :)
